core/src/main/java/org/jdeferred/ExceptCallback.java

```diff
@@ -0,0 +1,8 @@
+package org.jdeferred;
+
+/**
+ * Created by nwertzberger on 3/16/15.
+ */
+public interface ExceptCallback {
```

Comment:
The name "except" is related to "exception" but have different meanings. I'd prefer if this callback were to be named "ExceptionCallback".